wkcuber/downsampling.py

```diff
@@ -3,16 +3,14 @@
 import re
 import numpy as np
 from argparse import ArgumentParser
-from math import floor, log2
-from os import path, listdir
+import os
 from scipy.ndimage.interpolation import zoom
 from itertools import product
-from functools import lru_cache
 from enum import Enum
-from .mag import Mag
+from wkcuber.mag import Mag
```

Comment:
Why no relative imports?

Reply:
I'm having issues with the remote docker interpreter path. It does not really find the relative files. Using absolute imports are fixing the issue. I reverted the change.